packages/editor-ui/src/components/MainHeader/WorkflowDetails.vue

```diff
@@ -132,9 +128,9 @@ const onExecutionsTab = computed(() => {
 	].includes((route.name as string) || '');
 });
 
-const workflowPermissions = computed<PermissionsMap<WorkflowScope>>(() => {
-	return getWorkflowPermissions(workflowsStore.getWorkflowById(props.workflow.id));
-});
+const workflowPermissions = computed(
+	() => getResourcePermissions(workflowsStore.getWorkflowById(props.workflow.id)?.scopes).workflow,
```

Comment:
Can you extract the `workflow` ( `workflowsStore.getWorkflowById(props.workflow.id)` ) into a separate `computed` property? Otherwise the value won't be cached.